UnityProject/Assets/Scripts/Objects/Engineering/Reactor/ReactorBoiler.cs

```diff
@@ -29,6 +30,9 @@ public class ReactorBoiler : MonoBehaviour, IMultitoolMasterable, ICheckedIntera
 		public void Awake()
 		{
 			ReactorPipe = GetComponent<ReactorPipe>();
+			ReactorPipe.pipeData.mixAndVolume.SetVolume(10);
```

Comment:
should be able to set this in the editor, though when I allowed to be serialised apparently start zero so  it must get set somewhere and can't work out where so this is fine for now

Reply:
Done

Reply:
well as I meant as in going into pipeData in the editor and setting there , Though it's defaulting to 0 so I imagine it gets set somewhere to 1 idk where that is